src/main/java/pl/javastart/task/Ticket.java

```diff
@@ -0,0 +1,69 @@
+package pl.javastart.task;
+
+public class Ticket {
+    private String event;
+    private Adress adress;
+    private double price;
+    private String type;
+    static int ticketNo;
+
+    public Ticket(String event, Adress adress, double price, String type) {
+        this.event = event;
+        this.adress = adress;
+        this.price = price;
+        this.type = type;
+        ticketNo++;
+    }
+
+    public Ticket() {
+        ticketNo++;
+    }
```

Comment:
Aktualnie wszystkie bilety mają u Ciebie taki sam identyfikator. Jeżeli w metodzie main wyświetlisz id dla biletu ticket1 i ticket2, to będzie ta sama wartość.
Każdy bilet powinien mieć unikalne id, potrzebujesz tutaj jeszcze jednego pola, które nie będzie statyczne (to pole statyczne też będzie Ci jednak potrzebne).
To pole ticketNo powinno być też prywatne